src/lib/components/Header.svelte

```diff
@@ -50,17 +47,16 @@
 	<div class="flex-none space-x-2">
 		<ThemeToggle />
 		<Tour />
-		<div class="dropdown dropdown-end">
-			<div tabindex="-1" role="button" class="btn btn-ghost" onmousedown={checkAndCloseDropDown}>
```

Comment:
Replaced this whole section with the alternative set of HTML-Elements suported by daisyUI dropdown in favor of avoiding having `role="button"` on a div.